.github/workflows/test.yml

```diff
@@ -15,8 +15,8 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  GO_VERSION: "1.19.6"
-  K8S_VERSION: v1.24.0
+  GO_VERSION: "1.20.6"
+  K8S_VERSION: v1.28.0
```

Comment:
its because KIND-est node has not had a 1.28.3 release :( So except in the kind.install script everywhere else I have moved it to 1.28.3. Once upstream KIND releases a 1.28.3 version for the nodes will bump that as well, but good question.